.github/workflows/ibis-backends.yml

```diff
@@ -467,6 +438,73 @@ jobs:
         if: ${{ failure() }}
         run: docker compose logs
 
+  test_pyspark:
+    name: PySpark ${{ matrix.os }} python-${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+        python-version:
+          - "3.8"
+          - "3.10"
+          - "3.11"
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: microsoft
+          java-version: 17
+
+      - uses: extractions/setup-just@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: download backend data
+        run: just download-data
+
+      - name: install python
+        uses: actions/setup-python@v4
+        id: install_python
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - run: python -m pip install --upgrade pip 'poetry<1.4'
+
+      - name: install minimum versions
+        run: poetry add --lock 'pandas>=1.2.5,<2' 'numpy>=1,<1.24'
```

Comment:
Do we want to have a pyspark build with pandas 2?

Reply:
Probably not worth the trouble yet, we can revisit in a few months. It'll probably take a really long time for pyspark to support pandas 2.

Reply:
Hm, actually it's pretty easy to add this and it'll be nice to know when it's finally supported.

Reply:
Done.